.github/workflows/get-streamers.yml

```diff
@@ -16,3 +15,20 @@ jobs:
         run: npm install
       - name: Run script
         run: npm run getstreamers
+      - name: Commit changes
+        run: |
+          something_changed=`git diff-index --exit-code --ignore-submodules HEAD`
+          if [ -n "$something_changed" ]
+          then
+              echo "Something changed, committing"
+              git config user.name github-actions
+              git config user.email github-actions@github.com
+              git add .
+              git commit -m "get-streamers.js updated the streamers.json file"
+              git push
+              exit 0
+          else
+              echo "No changes, exiting"
+              exit 0
+          fi
+        shell: bash {0}
```

Comment:
This is needed to prevent the default "fail-fast" behaviour that would stop the script at line 20 when 'git giff-index' returns a non-zero return code.
Reference: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#exit-codes-and-error-action-preference